spec/components/tabs_spec.rb

```diff
@@ -31,7 +31,29 @@ def component_name
     assert_select "#tab1"
   end
 
-  it "renders with GA4 tracking" do
+  it "renders tabs as links" do
+    render_component(
+      as_links: true,
+      tabs: [
+        {
+          label: "First section",
+          href: "/page1",
+        },
+        {
+          label: "Second section",
+          href: "/page2",
+        },
+      ],
+    )
+
+    assert_select ".govuk-tabs"
+    assert_select ".govuk-tabs__tab", 2
+    assert_select ".govuk-tabs__list-item", 2
+    assert_select ".govuk-tabs__tab[href='/page1']"
+    assert_select ".govuk-tabs__tab[href='/page2']"
```

Comment:
Nitpick: Should this test check that they're actually `<a>` tags? E.g. `assert_select "a.govuk-tabs__tab[href='/page1']"`

Reply:
Good shout, have updated, thanks.